index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Microverse mobile project</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+    <header>
+            <section class="logo-menubar">
+                <h4 class="logo">
+                    <a href="#">
+                            My Logo
+                    </a>
+                </h4>
+                <nav>
+                    <img src="./img/Normal Button/Tertiary/Icons/Icon - Menu.svg" class="hamb-icon" alt="menu-bar">
+                </nav>
+            </section>
+    </header>
+    <section class="presentation-part">
+                    <h1>I'm Ivonne. Glad to see you!</h1>
+                    <p class="presentation-paragraph">I am a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+                    <div class="socialnetwork-part">
+                            <p class="socialtext-part">
+                                LET'S CONNECT
```

Comment:
- [optional] please kindly use an `h` tag here as it is a subtitle.

Reply:
Dear @tufoinnkuo10 
Very nice to meet you, I appreciate your feedback, thank you.
I will merge the project, 
Best regards,